crates/compiler/builtins/bitcode/src/crypto.zig

```diff
@@ -7,50 +7,45 @@ const utils = @import("utils.zig");
 const testing = std.testing;
 
 const Sha256 = extern struct {
-    location: *sha2.Sha256,
+    data: [@sizeOf(sha2.Sha256)]u8 align(@alignOf(sha2.Sha256)),
```

Comment:
I'm working on the assumption that the new point of panic was the call to `allocateWithRefcount`. I think this recreates the suggestion on @bhansconnect that I accidentally force-merged away.